File: frontend/src/app/interfaces/response/hotel.response.ts
export interface HotelResponse {
  id: number;
  code: string;
  name: string;

  cityId: number;
  cityName: string;

  countryId: number;
  countryIso2: string;
  countryName: string;

  address?: string;
  stars?: number;

  latitude?: number;
  longitude?: number;

  status: string;
}

File: backend/src/main/java/com/fdsa/altairis/dto/response/HotelResponse.java
package com.fdsa.altairis.dto.response;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.math.BigDecimal;

@Getter
@Setter
@ToString
public class HotelResponse {

    private Long id;
    private String code;
    private String name;

    private Long cityId;
    private String cityName;

    private Long countryId;
    private String countryIso2;
    private String countryName;

    private String address;
    private Integer stars;

    private BigDecimal latitude;
    private BigDecimal longitude;

    private String status;

}
